crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -876,7 +877,7 @@ mod tests {
     /// Return a full path to the file with `file_name`.
     fn format_path(file_name: &str) -> String {
         format!(
-            "{COMPRESSED_DATA_FOLDER}/{TABLE_NAME}/{COLUMN_INDEX}/{file_name}_{}.parquet",
+            "{COMPRESSED_DATA_FOLDER}/{TABLE_NAME}/{COLUMN_INDEX}/{file_name}_{}_{PORT}.parquet",
```

Comment:
The `const`s should be referenced the same way, so either imported completely like `PORT` or by importing the module like `storage::TEST_UUID`.

Reply:
Importing them both completely to make the `format!` cleaner.